### ISSUE TITLE
What is distillation loss in the code used for ? 

### ISSUE THREAD

**Issue by wqshmzh** [snumprlab/cl-alfred#8]
I notice that the distillation loss in your code is not mentioned in your paper. So what are they used for ?

**Comment by bhkim94** [comment]
Hi @wqshmzh,

The distillation loss is $\alpha \mathbb{E}_{(x,z)\sim\mathcal{M}}[||z - \pi (x)||^2_2]$ of Equation 3 in our paper, which corresponds to [L153-L160](https://github.com/snumprlab/cl-alfred/blob/acafe80a75d2bbb45a5157e7e794c99be71fc0f0/methods/cama.py#L153). The current model (*i.e.*, learning the current task) is trained with the distillation loss such that the generated logits get close to ones generated from the previous models (*i.e.*, learning the previous tasks).

**Comment by wqshmzh** [comment]
OK. Got it. Sorry my bad !
